src/components/common/Typography/Form/FormFields/UrlField/UrlField.jsx

```diff
@@ -58,6 +61,13 @@ const UrlField = (props) => {
     let value = event ? event.target.value : '';
     if (isEmpty(value)) {
       onChange();
+    } else if (unsecured) {
+      if (value.includes('https://') || value.includes('http://')) {
```

Comment:
Simplify this condition 
```suggestion
      if (!value.includes('https://') || !value.includes('http://')) {
        value = `http://${value}`;
      }
```

Reply:
Done